pkg/sentry/server/server.go

```diff
@@ -102,6 +102,10 @@ func (s *server) getServerCertificate() (*tls.Certificate, error) {
 	issuerExp := s.certAuth.GetCACertBundle().GetIssuerCertExpiry()
 	serverCertTTL := issuerExp.Sub(now)
 
+	if serverCertTTL <= 30 {
```

Comment:
What is `30` here? is it supposed to be days? If it's days, then:

1. It won't work as the value of `time.Duration` doesn't represent days
2. the error message should be "The root CA cert is about to expire in %s days"

Reply:
Also, please add a test for this.